docs/user/metrics/github.csv

```diff
@@ -7,3 +7,4 @@ Library,Contributors,Forks,Watchers,Stars,Issues,PRs,Releases,Commits,Core SLOCs
 `DeepForest`_,17,172,17,474,413,301,44,864,"3,357","1,794",86%,MIT
 `SITS`_,14,76,28,451,622,583,44,"6,244","24,284","8,697",94%,GPL-2.0
 `TerraTorch`_,9,10,9,121,46,92,2,243,"10,101",583,44%,Apache-2.0
+`scikit-eo`_,6,17,8,132,20,11,15,496,,,Apache-2.0
```

Comment:
@yotarazona should be able to answer here as well

Reply:
@adamjstewart and @Nowosad, the "shp" argument has been changed to "vector" since some versions ago. Now you should not get errors. About the DeepLearning test error, it's weird but I don't get any error :).

Reply:
I would highly recommend running pytest in CI, it's the only way to prevent bugs like this from appearing. Let me know if you need any help setting up CI, and feel free to use TorchGeo's CI as an example.